frontend/src/views/SessionPage.vue

```diff
@@ -477,6 +476,7 @@ export default defineComponent({
     }
     if (this.voteSetJson) {
       this.voteSet = JSON.parse(this.voteSetJson);
+      this.voteSet = ["?"].concat(this.voteSet);
```

Comment:
Do we need to concat the '?' to our voteset here, or would it be better to directly set the '?' as an activeValue for the allCardSetsWithJiraMode in the CardSetComponent.vue ?